generate-docs/script-inputs/local-repo-snippets.yaml

```
@@ -1798,6 +1798,39 @@ Office.AddinCommands.Source#id:member:
 
         event.completed();
     }
+Office.AsyncContextOptions:interface:
+  - |-
+    // The following example gets the attachment contents of the
+    // current mail item being composed in Outlook.
+    function getAttachmentContentCompose() {
+        const item = Office.context.mailbox.item;
+        const options = { asyncContext: { currentItem: item } };
```

For these class-level samples, we should make it very clear what the object is.
```suggestion
        const options: Office.AsyncContextOptions = { asyncContext: { currentItem: item } };
```